src/lib/components/inbox/bids/active_bid.tsx

```diff
@@ -0,0 +1,168 @@
+  height: 1;
+  width: 100%;
+  background-color: ${colors["gray-regular"]};
+  margin-top: 17px;
+`
+
+const StatusLabel = styled(MetadataText) `
+  color: ${props => {
+    switch (props.status) {
+      case "winning":
+        return colors["green-regular"]
+      case "reserve":
+        return colors["yellow-regular"]
+      case "losing":
+        return colors["red-regular"]
+    }
+  }};
+`
+
+type BidStatus = "winning" | "reserve" | "losing"
+
+interface State {
+  status: BidStatus
+  statusLabel: string
+}
+
+class ActiveBid extends React.Component<any, State> {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      status: "losing",
+    }
+  }
+
+  componentDidMount() {
+    this.updateStatus()
+  }
+
+  updateStatus() {
+    const bid = this.props.bid
+    const leadingBidder = bid.is_leading_bidder
+    const reserveNotMet = bid.active_bid.sale_artwork.reserve_status === "reserve_not_met"
+
+    let status = "losing"
+    if (leadingBidder) {
+      status = reserveNotMet ? "reserve" : "winning"
+    }
+
+    this.setState({ status })
+  }
+
+  get statusLabel(): string {
+    switch (this.state.status) {
+      case "winning":
+      case "reserve":
+        return "Highest Bid"
+      case "losing":
+        return "Outbid"
+    }
+  }
+
+  render() {
+    const bid = this.props.bid.active_bid
+    const imageURL = bid.sale_artwork.artwork.image.url
+    const lotNumber = bid.sale_artwork.lot_number
+    const artistName = bid.sale_artwork.artwork.artist.name
+
+    const headline = `Lot ${lotNumber} · ${artistName} `
+    const subtitle = `Current Bid: ${bid.max_bid.display} `
+
+    return (
+      <TouchableWithoutFeedback>
+        <Container>
+          <Content>
+            <ImageView imageURL={imageURL} />
+            <MetadataContainer>
+              <BodyText>{headline}</BodyText>
+              <BodyText>{subtitle}</BodyText>
+            </MetadataContainer>
+            <StatusLabel status={this.state.status}>{this.statusLabel}</StatusLabel>
+          </Content>
+          <Separator />
+        </Container>
+      </TouchableWithoutFeedback>
+    )
+  }
+}
+
+export default Relay.createContainer(ActiveBid, {
+  fragments: {
+    bid: () => Relay.QL`
+      fragment on LotStanding {
+        is_leading_bidder
+        active_bid {
+          id
+          max_bid {
+            cents
+            display
+          }
+          sale_artwork {
+            lot_label
+            lot_number
+            position
+            reserve_status
+            counts {
+              bidder_positions
+            }
+            sale {
+              live_start_at
+              end_at
+              is_live_open
+              is_closed
+            }
+            highest_bid {
+              cents
+              display
+            }
+            artwork {
+              id
+              title
+              image {
+                url
+              }
+              artist {
```

Comment:
This should be `artist_names` - this will not account for artworks with multiple artists and metaphysics does the string formatting for us already.